### ISSUE TITLE
[Feature Request]: Customize Deal Tags: Payments Pipeline

### ISSUE THREAD

**Issue by laneymangan** [cal-itp/customer-success#271]
### Feature Request

We recently started utilizing Hubspot's tagging feature to help categorize and sort tickets and deals in our various pipelines. We would like to ensure that custom tagging automations are being utilized in all pipelines where they would be useful. We would like to activate this feature for the `Payments Pipeline`

### Is this request related to a problem?

Pipelines can quickly become overwhelming, the hope is that this tagging system will help our pipelines stay organized and cut down on time spent sorting through various stages and deals/tickets.

### Use Case

We recently set up this feature in our internal support pipeline to differentiate between CRM Maintenance tickets and GTFS Data Quality tickets. This feature has been well-received and we would like to do something similar in our Payments Pipeline

### Additional Info

@o-ram's suggestions (copied from ticket #256)

> Payments pipeline ideas:
> (@laneymangan please feel free to add to this! I went with regular bullet points instead of a checkmark list because these ideas are a lot less formulated and very rough brainstorming)
> 
> Something with latest status property?
> Something with Felix property?
> Something related to names - in-state vs. out of state, district number
> "critical"
> TAP/Clipper designations?
> something related to certain period of time of inaction? (like a stale deal)
> 
> Shown in the Thursday AM meeting to those who were there until the end, Erika to think through any ideas she has and share them with Laney and/or Olivia.

**Comment by AnthonyRollins** [comment]
@laneymangan do we have enough deal tags for this to be a viable option for the payments pipeline?

**Comment by o-ram** [comment]
@AnthonyRollins yes, we have deal tags and Laney and I talked about this ticket a bit. She is assigned to it.

**Comment by laneymangan** [comment]
@o-ram are all of these properties essential to the board view for the payments pipeline? can any be removed?

- Latest Status
- TAP Agency
- Agency Signed Contracts (maybe this one can be replaced with the new property which 
- Felix Demo

![image](https://github.com/cal-itp/customer-success/assets/107429473/0d8ffa67-7498-4186-bdf3-05da3dc92426)

**Comment by o-ram** [comment]
@laneymangan the only one of those that should stay is `Agency signed contracts`. The rest can go!

**Comment by laneymangan** [comment]
I added "Implementation" and "Contracts" as multi-select deal properties that are displayed in the card view. 
<img width="262" alt="image" src="https://github.com/cal-itp/customer-success/assets/107429473/3e83316f-2b17-4935-a47c-a249c6d5c61c">

**Comment by laneymangan** [comment]
@o-ram should I go ahead and add "ready for benefits" to the payments pipeline? This could be a deal tag or a property.

**Comment by laneymangan** [comment]
> @laneymangan the only one of those that should stay is `Agency signed contracts`. The rest can go!

@o-ram I removed the rest from the card view. Should these also be removed from the payments pipeline conditional section within the ticket?
![image](https://github.com/cal-itp/customer-success/assets/107429473/e9923f64-6a49-4604-9be5-0892d185efb5)

**Comment by laneymangan** [comment]
`Agency signed contracts` and `contracts` have been removed

**Comment by o-ram** [comment]
Thanks, Laney! I started playing around with adding the vendors via association to replace the removed properties. Grabbing some time for us to chat!
